src/test/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcPageHeadingSpec.scala

```diff
@@ -0,0 +1,309 @@
+            + """Personal details"""
+            + """</span>"""
+        ),
+        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2"
+      )
+    }
+
+    "accept custom classes and attributes" in {
+      HmrcPageHeadingLabel(
+        text = "What is your name?",
+        section = "Personal details",
+        classes = "foo bar",
+        attributes = Map("foo" -> "bar")
+      ) mustBe Label(
+        isPageHeading = true,
+        content = HtmlContent(
+          """What is your name? """
+            + """<span class="govuk-caption-xl hmrc-caption-xl">"""
+            + """<span class="govuk-visually-hidden">This section is (english translation) </span>"""
+            + """Personal details"""
+            + """</span>"""
+        ),
+        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2 foo bar",
+        attributes = Map("foo" -> "bar")
+      )
+    }
+
+    "translate the default visually hidden text" in {
+      HmrcPageHeadingLabel(
+        text = "What is your name?",
+        section = "Personal details",
+        visuallyHiddenText = Some("This section is")
+      ) mustBe Label(
+        isPageHeading = true,
+        content = HtmlContent(
+          """What is your name? """
+            + """<span class="govuk-caption-xl hmrc-caption-xl">"""
+            + """<span class="govuk-visually-hidden">This section is (english translation) </span>"""
+            + """Personal details"""
+            + """</span>"""
+        ),
+        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2"
+      )
+    }
+
+    "allow you to override the default visually hidden text" in {
+      HmrcPageHeadingLabel(
+        text = "What is your name?",
+        section = "Personal details",
+        visuallyHiddenText = Some("Section")
+      ) mustBe Label(
+        isPageHeading = true,
+        content = HtmlContent(
+          """What is your name? """
+            + """<span class="govuk-caption-xl hmrc-caption-xl">"""
+            + """<span class="govuk-visually-hidden">Section </span>"""
+            + """Personal details"""
+            + """</span>"""
+        ),
+        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2"
+      )
+    }
+
+    "allow you to omit the visually hidden text" in {
+      HmrcPageHeadingLabel(
+        text = "What is your name?",
+        section = "Personal details",
+        visuallyHiddenText = None
+      ) mustBe Label(
+        isPageHeading = true,
+        content = HtmlContent(
+          """What is your name? """
+            + """<span class="govuk-caption-xl hmrc-caption-xl">"""
+            + """Personal details"""
+            + """</span>"""
+        ),
+        classes = "govuk-label--xl hmrc-page-heading govuk-!-margin-top-0 govuk-!-margin-bottom-2"
+      )
+    }
+
+    "escape unsafe html output" in {
```

Comment:
Should we add some unsafe HTML here?

Reply:
@matthewmascord thank you, I forgot to update the test after updating the title 🤦‍♂️